### DISCUSSION TITLE
Tooltip in high latency mode

### DISCUSSION THREAD

**Discussion by datasynergyuk** [jovibor/ListEx#3] [Ideas]
It would be useful if there was a tooltip displayed when operating in high latency mode. This could be something like "Record 6/N" or perhaps configured via a custom format string or callback.

**Comment by jovibor** [comment]
Implemented in 5070255.

**Reply by datasynergyuk** [reply]
Thank you very much.

Please can I ask what the POINT field in the LISTEXHDRICON structure is for. It doesn't seem to be used yet?

**Reply by jovibor** [reply]
It's stated in the description:
`POINT pt { }; //Point of the top-left corner.`
Icon's top left corner coord.
By default it's 0,0.
